doc/nrf/ug_contribution.rst

```diff
@@ -0,0 +1,142 @@
+.. _ug_contribution:
+
+Contributing to the |NCS|
+*************************
+
+The |NCS| is maintained by Nordic Semiconductor, but it is open to community contributions.
+
+As a general practice, we encourage to make small, controlled changes.
+This practice simplifies reviewing, merging, and rebasing.
+In addition, it keeps the change history clear and clean.
+
+When contributing to the |NCS|, you should provide as much information as you can about your change.
+Before submitting, you must also update the appropriate documentation and test your changes thoroughly.
+
+The general GitHub workflow that is used by |NCS| developers is based on a combination of Git commands on the command line and browser interaction with GitHub.
+As usual with Git, there are multiple ways of getting a task done.
+The following steps show a typical workflow:
+
+1. :ref:`dm-wf-fork`.
+   The following substeps summarize the most important commands to start working with a fork of the repository:
+
+   a. Fork the fw-nrfconnect-nrf repository to your personal account on GitHub.
+      To do so, click the `fork button on the fw-nrfconnect-nrf repository page`_ (upper right corner).
+
+   #. On your computer, change into the :file:`nrf` folder that was created when you :ref:`obtained the code <cloning_the_repositories_win>`::
+
+         cd ncs/nrf
+
+      Rename the default remote that points to the `upstream nRF Connect SDK repository <fw-nrfconnect-nrf_>`_ from ``origin`` to ``ncs``::
+
+         git remote rename origin ncs
+
+      Let Git know about the fork that you just created and name it ``origin``.
+      Replace *your_github_id* with your GitHub user name.
+
+      .. parsed-literal::
+         :class: highlight
+
+         git remote add origin https\://github.com/*your_github_id*/fw-nrfconnect-nrf
+
+      Finally, verify the remote repositories::
+
+         git remote -v
+
+      The output should look similar to this:
+
+      .. parsed-literal::
+         :class: highlight
+
+         origin   https\://github.com/*your_github_id*/fw-nrfconnect-nrf (fetch)
+         origin   https\://github.com/*your_github_id*/fw-nrfconnect-nrf (push)
+         ncs      https\://github.com/NordicPlayground/fw-nrfconnect-nrf (fetch)
+         ncs      https\://github.com/NordicPlayground/fw-nrfconnect-nrf (push)
+
+#. Create a topic branch (based on the master branch) for your work.
+   Replace *branch_name* with the name that you want to use for the local branch.
+
+   .. parsed-literal::
+      :class: highlight
+
+      git checkout -b *branch_name* ncs/master
+
+   If you are addressing a specific issue, we suggest including the issue number in the branch name.
+
+#. Make changes, test locally, change, test, test again, ...
+   Check out the *sanitycheck* section in Zephyr's :ref:`zephyr:Contribution Tools` documentation as well.
```

Comment:
Does it make sense to mention sanitycheck? And if so, is this the correct link? (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/contribute/index.html?highlight=sanitycheck#sanitycheck)